<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordTabWidget</class>
 <widget class="QWidget" name="RecordTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RecordTabWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="recordingBox">
     <property name="title">
      <string>Recroding</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="layoutRecording">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="layoutRecordButtons">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QPushButton" name="buttonRecord">
            <property name="text">
             <string>Record</string>
            </property>
            <property name="icon">
             <iconset resource="InputRepeater.qrc">
              <normaloff>:/Resources/RecordIcon.png</normaloff>:/Resources/RecordIcon.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonStop">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Stop</string>
            </property>
            <property name="icon">
             <iconset resource="InputRepeater.qrc">
              <normaloff>:/Resources/StopIcon.png</normaloff>:/Resources/StopIcon.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="inputTypeBox">
          <property name="title">
           <string>Input type</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QRadioButton" name="radioKeyboard">
             <property name="text">
              <string>Keyboard</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioMouse">
             <property name="text">
              <string>Mouse</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioKeyboardAndMouse">
             <property name="text">
              <string>Keyboard and mouse</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="QGroupBox" name="infoBox">
        <property name="title">
         <string>Info</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QFormLayout" name="layoutInfo">
           <item row="0" column="0">
            <widget class="QLabel" name="labelRecordStatusTitle">
             <property name="text">
              <string>Record status:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="labelRecordStatus">
             <property name="text">
              <string>Unloaded</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="labelFilenameTitle">
             <property name="text">
              <string>Filename:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="editFilename">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="labelFullPathTitle">
             <property name="text">
              <string>Full path</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="editFullPath">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="buttonUnload">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Unload</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelTypeTitle">
             <property name="text">
              <string>Type:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="labelType">
             <property name="text">
              <string>None</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="layoutFileButtons">
           <item>
            <widget class="QPushButton" name="buttonSaveAs">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Save as...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonLoadFrom">
             <property name="text">
              <string>Load from...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonTransform">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Transform</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="InputRepeater.qrc"/>
 </resources>
 <connections/>
</ui>
